data/pigui/modules/hyperjump-planner.lua

```diff
@@ -103,28 +101,28 @@ local function showInfo()
 	end
 end -- showInfo
 
-local function mainButton(icon, tooltip, callback)
-	local button = ui.coloredSelectedIconButton(icon, mainButtonSize, false, mainButtonFramePadding, colors.buttonBlue, colors.buttonInk, tooltip)
+local function smallButton(icon, tooltip, callback)
+	local button = ui.mainMenuButton(icon, tooltip, nil, ui.theme.styles.SmallButtonSize)
 	if button then
 		callback()
 	end
 	return button
-end --mainButton
+end -- smallButton
 
 local function buildJumpRouteList()
 	hyperjump_route = {}
-	local player = Game.player
+	local plr = Game.player
 	-- if we are not in the system, then we are in hyperspace, we start building the route from the jump target
-	local start = Game.system and Game.system.path or player:GetHyperspaceDestination()
-	local drive = table.unpack(player:GetEquip("engine")) or nil
+	local start = Game.system and Game.system.path or plr:GetHyperspaceDestination()
+	local drive = table.unpack(plr:GetEquip("engine")) or nil
 	local fuel_type = drive and drive.fuel or Equipment.cargo.hydrogen
-	local current_fuel = player:CountEquip(fuel_type,"cargo")
+	local cur_fuel = plr:CountEquip(fuel_type,"cargo")
```

Comment:
I'm not sure why you've changed `player` here to `plr` - it's only a three character difference, and the variable name doesn't gain any clarity from dropping all of the vowels; to me, it's actually harder to tell what this variable is referring to.